migration/sql-files/014-prevent-invalid-work-item-links.sql

```diff
@@ -0,0 +1,102 @@
+-- is_type_or_subtype_of returns true if the typeName is of the given witName,
+-- or a subtype according to the witPath; otherwise false is returned.
+CREATE OR REPLACE FUNCTION is_type_or_subtype_of(typeName TEXT, witName TEXT, witPath TEXT)
```

Comment:
This is a bad attempt to solve the problem. Here's a better one: https://github.com/almighty/almighty-core/pull/602/files#diff-bb6237dff93157113040a5bb1d9859c5R3